lhotse/dataset/speech_recognition.py

```diff
@@ -32,7 +32,7 @@ def __init__(
             root_dir: Optional[Pathlike] = None
     ):
         super().__init__()
-        self.cuts = cuts.cuts
+        self.cuts = cuts.trim_to_supervisions().cuts
```

Comment:
```suggestion
        self.cuts = cuts.trim_to_supervisions()
```

I think the `.cuts` at the end is not needed - `CutSet` works almost like a dict already.